pgmpy/models/BayesianModel.py

```diff
@@ -738,6 +747,72 @@ def predict(self, data):
                 pred_values[k].append(v)
         return pd.DataFrame(pred_values, index=data.index)
 
+    def predict_probability(self, data):
```

Comment:
@ankurankan So you suggest a single method that calls two methods(predict and predict_probability) based on a parameter.  Please correct me if I am getting this wrong.